### DISCUSSION TITLE
chrome://kill not allowed in policy :((( >:((( 😭😭😭😭

### DISCUSSION THREAD

**Discussion by ILikeBypassing** [3kh0/ext-remover#659] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
hi

chrome://crash
chrome://memory-exhaust
chrome://badcastcrash

**Comment by deryilz** [comment]
wow someone who knows what theyre talking about !?

**Comment by ghost** [comment]
Hello. You can still keep GoGuardian from working by temp. crashing it (file perm switch in extension page or by task manager) which clears its cache. From there you could chrome://hang its manifest to prevent it from reading policies. Hope this helped.
